.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Bug report
+about: Bug
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**What is wrong**
+
+
+**How should it be**
```

Comment:
i would prefer expected behaviour and current behaviour
an area for Android Logcat output would be useful